etc/functions/internal.json

```diff
@@ -3387,6 +3387,29 @@
 				"zh_CN": "时间戳"
 			}
 		}
+	}, {
+		"name": "rule_id",
+		"example": "rule_id()",
+		"hint": {
+			"en_US": "Returns the ID of the currently matched rule",
+			"zh_CN": "返回当前匹配到的规则的ID。"
+		},
+		"args": [],
+		"return": {
+			"type": "string",
+			"hint": {
+				"en_US": "ID of currently matched rule",
+				"zh_CN": "当前匹配到的规则的ID"
+			}
+		},
+		"node": {
+			"category": "function",
+			"icon": "iconPath",
+			"label": {
+				"en_US": "ID of currently matched rule",
```

Comment:
Label is the word to show in UI, should be short. Suggest "Rule ID"

Reply:
done